tests/e2e/zookeeper_test.go

```diff
@@ -92,9 +118,9 @@ func requireRemoveZookeeperOperatorCRDs(kubectlOptions *k8s.KubectlOptions) {
 func requireZookeeperClusterReady(kubectlOptions *k8s.KubectlOptions) {
 	It("Verifying Zookeeper cluster health", func() {
 		By("Verifying the Zookeeper cluster resource")
-		waitK8sResourceCondition(kubectlOptions, zookeeperCRDs()[0], "jsonpath={.status.readyReplicas}=1", "240s", "", zookeeperClusterName)
+		waitK8sResourceCondition(kubectlOptions, zookeeperCRDs()[0], "jsonpath={.status.readyReplicas}=1", zookeeperClusterCreateTimeout, "", zookeeperClusterName)
 		By("Verifying the Zookeeper cluster's pods")
-		waitK8sResourceCondition(kubectlOptions, "pod", "condition=Ready", "60s", "app="+zookeeperClusterName, zookeeperClusterName)
+		waitK8sResourceCondition(kubectlOptions, "pod", "condition=Ready", defaultPodReadinessWaitTime, "app="+zookeeperClusterName, zookeeperClusterName)
```

Comment:
We should only use the selector here. In case when multiple zookeeper server is present we can check all of readiness.
We should not use selector and name altogether for waitK8sResourceCondition and getK8sResources

Reply:
I mean, why not? Here if we have multiple zookeeper clusters do we really care about all of them or just the one that we are trying to deploy right now? I feel it should be okay to only check the one we are deploying instead of all of them, but I may be misunderstanding something.

Reply:
The selector "app="+zookeeperClusterName  selects one zookeeper cluster. I think this check is not working right now but I have not tested. It seems that you want to check a pod readiness and select the pod with selector and with a zookeeperClusterName which is wrong because it is not a pod name.
The only reason for that is work because I wrote this function in this way: https://github.com/banzaicloud/koperator/blob/ef6db9416dc459e992587ad259c00da1ac2fa130/tests/e2e/utils.go#L53 :)

Reply:
When you using names then you need to specify the resource exact name so the selector is kinda useless.
kubectl does not support both (error: name cannot be provided when a selector is specified)